.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: python
+dist: xenial
 python:
-  - "3.6"
-  - "3.5"
+    - "3.7"
```

Comment:
🎨 dunno if intended

Reply:
Other sdx repos seem to be the same, though I'm not sure why...  I'll leave it in for consistency